client/my-sites/earn/home.tsx

```diff
@@ -195,7 +195,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		}
 
 		const cta: CtaButton = {
-			text: translate( 'Earn Free Credits' ) as string,
+			text: translate( 'Earn free credits' ) as string,
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.